src/typography/display2/specifications/display2.md

```diff
@@ -0,0 +1,34 @@
+# TextDisplay2
+
+`TextDisplay2` is text component on top of `Text`.
+
+<!-- STORY -->
+
+## Usage
+
+```js
+import TextDisplay2 from 'typography/display2'
+
+<TextDisplay2>The quick brown fox jumps over the lazy dog</TextDisplay2>
+```
+
+## Rendering
+
+
+| **Weight**   | **Size** | **Line**          | **Color** |
+| ------------ | -------- | ----------------- |-----------|
+| Medium (500) | 82px     | Height 82px (1.0) | title \| white |
+
+`TextDisplay2` is text element `<span />` and **should receive its semantics by its parent** as you can see on the example bellow
+
+```js
+<h1>
+  <TextDisplay2>The quick brown fox jumps over the lazy dog</TextDisplay2>
+</h1>
+```
+
+## Props
+
+| **Props**    | **Value**       | **Description**                  |
+| ------------ | --------------- | -------------------------------- |
+| `isInverted` | `true`\|`false` | When applied to dark backgrounds |
```

Comment:
Thanks! This is nice, I'm ok to go with that for now, but we should probably think on having a documentation template so that all the documentation section are in the same order. Also I'm not a big fan of having `Props` section since we're using typescript it's easy enough to use it, and if it's not we should look into a way to generate it automatically in the docs otherwise it will get out of date really quick.

Reply:
A generated list could be nice, but we really need these descriptions for each props. Thanks @gabrielagabriel for adding this documentation.